### ISSUE TITLE
VSOD数据集

### ISSUE THREAD

**Issue by typist2001** [Sssssuperior/VSCode#11]
您好，请问vsod数据集中 FBMS, ViSal, SegV2等直接下载没有Frame和OF_FlowNet2这两个文件，无法用于测试。请问是用自己用[Flownet2.0]去创建吗？或者可以上传一下现成的吗？

**Comment by Sssssuperior** [comment]
是用Flownet2.0创建的，直接的文件：https://pan.baidu.com/s/1c_KHJDI0GsVBlZA3bE_V-A, PIN: 1ezs

**Comment by typist2001** [comment]
真的很感谢，请问DAVSOD-easy-30+DAVSOD-Normal-25+DAVSOD-Difficult-20这些也有现成的文件吗

**Comment by Sssssuperior** [comment]
https://pan.baidu.com/s/1cHw7xaWwaMSHoiNWh7U1CQ, PIN: m3ij

**Comment by typist2001** [comment]
真的太感谢了

**Comment by CHULOKYEE** [comment]
Thanks for your excellent work. Would you please provide the test set of DAVIS, the link given in the README don't have the Frame and OF_FlowNet2. thx!

**Comment by Sssssuperior** [comment]
> 感谢您的出色工作。您能否提供 DAVIS 的测试集，README 中给出的链接没有 Frame 和 OF_FlowNet2。感谢！

You can download the dataset from: https://drive.google.com/file/d/1jOA6XOQ8-ZPXynOm1rMHv_Wt6PRduJmE/view?usp=sharing

**Comment by Sssssuperior** [comment]
> 至于视频任务，您是否在 DAVOS 数据集（使用 img、gt、flow 和 contour）上进行训练并在 DAVIS 数据集（使用 frame、gt 和 OF_FlowNet2）上进行测试？

Yes. We employ the following datasets to train our model concurrently: the training set of DUTS for RGB SOD , the training sets of NJUD, NLPR, and DUTLF-Depth for RGB-D SOD , the training set of VT5000 for RGB-T SOD , the training sets of DAVIS and DAVSOD for VSOD , the training sets of COD10K and CAMO for RGB COD , and the training set of MoCA-Mask for VCOD .

**Comment by CHULOKYEE** [comment]
> > 至于视频任务，您是否在 DAVOS 数据集（使用 img、gt、flow 和 contour）上进行训练并在 DAVIS 数据集（使用 frame、gt 和 OF_FlowNet2）上进行测试？
> 
> Yes. We employ the following datasets to train our model concurrently: the training set of DUTS for RGB SOD , the training sets of NJUD, NLPR, and DUTLF-Depth for RGB-D SOD , the training set of VT5000 for RGB-T SOD , the training sets of DAVIS and DAVSOD for VSOD , the training sets of COD10K and CAMO for RGB COD , and the training set of MoCA-Mask for VCOD .

Thanks for your prompt reply! If I use DAVOS as the training set (with img, gt, flow and contour), which dataset should I use for testing? I am a little confused.

**Comment by Sssssuperior** [comment]
The DAVSOD dataset is divided into a training set, validation set, and test set. The test set is further categorized into EASY-35, NORMAL-25, and DIFFICULT-20. We used the DAVSOD training set and evaluated on the EASY-35 and NORMAL-25 subsets. For more details, please refer to: https://github.com/DengPingFan/DAVSOD.

**Comment by CHULOKYEE** [comment]
Thanks for your patience! Would you please upload the link for DAVSOD training set with contour and optimal flow? The link on the baidu seems to be damaged. Can you upload the link for DAVSOD training set with contour and optimal flow by google drive? thx!!!

**Comment by Sssssuperior** [comment]
> Thanks for your patience! Would you please upload the link for DAVSOD training set with contour and optimal flow? The link on the baidu seems to be damaged. Can you upload the link for DAVSOD training set with contour and optimal flow by google drive? thx!!!

https://drive.google.com/file/d/1ttb91Q5LzryPyV8uHwkTHmLLtynNk1pk/view?usp=sharing

**Comment by Sssssuperior** [comment]
> Thanks for your patience! Would you please upload the link for DAVSOD training set with contour and optimal flow? The link on the baidu seems to be damaged. Can you upload the link for DAVSOD training set with contour and optimal flow by google drive? thx!!!

https://drive.google.com/file/d/1ttb91Q5LzryPyV8uHwkTHmLLtynNk1pk/view?usp=sharing

**Comment by CHULOKYEE** [comment]
> > Thanks for your patience! Would you please upload the link for DAVSOD training set with contour and optimal flow? The link on the baidu seems to be damaged. Can you upload the link for DAVSOD training set with contour and optimal flow by google drive? thx!!!
> 
> https://drive.google.com/file/d/1ttb91Q5LzryPyV8uHwkTHmLLtynNk1pk/view?usp=sharing

This link for the DAVSOD training set seems to be damaged as well. Could you please send another google drive link?

**Comment by Sssssuperior** [comment]
> The last link you provided for the DAVSOD training set seems to be damaged as well. Could you please send another google drive link? thx!!!
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "Sssssuperior/VSCode" ***@***.***&gt;; 发送时间:&nbsp;2024年9月26日(星期四) 晚上10:20 ***@***.***&gt;; 抄送:&nbsp;"Zhu ***@***.******@***.***&gt;; 主题:&nbsp;Re: [Sssssuperior/VSCode] VSOD数据集 (Issue #11) Thanks for your patience! Would you please upload the link for DAVSOD training set with contour and optimal flow? The link on the baidu seems to be damaged. Can you upload the link for DAVSOD training set with contour and optimal flow by google drive? thx!!! https://drive.google.com/file/d/1ttb91Q5LzryPyV8uHwkTHmLLtynNk1pk/view?usp=sharing — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***&gt;

In fact, I downloaded the provided dataset on my local computer, and there were no errors with the files... You can try using the unzip repair command. It might be an issue with the file format. I'll try packaging it into a new file, and you can see if this one works: https://drive.google.com/file/d/1wU0V8ePent79PZn4NtdBYreUcKvHju7b/view?usp=sharing

**Comment by CHULOKYEE** [comment]
> > The last link you provided for the DAVSOD training set seems to be damaged as well. Could you please send another google drive link? thx!!!
> > […](#)
> > ------------------ 原始邮件 ------------------ 发件人: "Sssssuperior/VSCode" _**@**_._**>; 发送时间: 2024年9月26日(星期四) 晚上10:20 _**@**_.**_>; 抄送: "Zhu _**@**_._**_**@**_.**_>; 主题: Re: [Sssssuperior/VSCode] VSOD数据集 (Issue #11) Thanks for your patience! Would you please upload the link for DAVSOD training set with contour and optimal flow? The link on the baidu seems to be damaged. Can you upload the link for DAVSOD training set with contour and optimal flow by google drive? thx!!! https://drive.google.com/file/d/1ttb91Q5LzryPyV8uHwkTHmLLtynNk1pk/view?usp=sharing — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: _**@**_.***>
> 
> In fact, I downloaded the provided dataset on my local computer, and there were no errors with the files... You can try using the unzip repair command. It might be an issue with the file format. I'll try packaging it into a new file, and you can see if this one works: https://drive.google.com/file/d/1wU0V8ePent79PZn4NtdBYreUcKvHju7b/view?usp=sharing

Thanks for your patience!
It seems that the link does not unzip properly in Linux systems, while it works fine when I download it through Windows.

**Comment by Zhaohy9346** [comment]
> > 您为 DAVSOD 训练集提供的最后一个链接似乎也已损坏。您能否发送另一个 google drive 链接？感谢！！！ [...](#) ------------------原始邮件 ------------------ 发件人：“Sssssuperior/VSCode” _**@**_。_**>; 发送时间： 2024年9月26日（星期四） 晚上10：20 _**@**_.**_>;抄送： “Zhu _**@**_._**_**@**_。**_>;主题： Re： [Sssssuperior/VSCode] VSOD数据集 （Issue [#11](https://github.com/Sssssuperior/VSCode/issues/11)） 感谢您的耐心等待！您能否上传带有轮廓和最佳流动的 DAVSOD 训练集的链接？百度上的链接似乎被破坏了。您可以通过 google drive 上传带有轮廓和最佳流的 DAVSOD 训练集的链接吗？感谢！！！https://drive.google.com/file/d/1ttb91Q5LzryPyV8uHwkTHmLLtynNk1pk/view?usp=sharing – 直接回复此电子邮件、在 GitHub 上查看或取消订阅。您收到此消息是因为您发表了评论。消息 ID：_**@**_.***>
> 
> 事实上，我在本地计算机上下载了提供的数据集，文件没有错误......您可以尝试使用 unzip repair 命令。这可能是文件格式的问题。我将尝试将其打包到一个新文件中，您可以查看此文件是否有效：https://drive.google.com/file/d/1wU0V8ePent79PZn4NtdBYreUcKvHju7b/view?usp=sharing

您好，您提供的这个 DAVSOD 训练集链接似乎已经损坏。您能否发送一个可用的Google或百度链接？感谢！！！ @Sssssuperior

**Comment by Sssssuperior** [comment]
> > > 您为 DAVSOD 训练集提供的最后一个链接似乎也已损坏。您能否发送另一个 google drive 链接？感谢！！！ [...](#) ------------------原始邮件 ------------------ 发件人：“Sssssuperior/VSCode” _**@**_。_**>; 发送时间： 2024年9月26日（星期四） 晚上10：20 _**@**_.**_>;抄送： “Zhu _**@**_._**_**@**_。**_>;主题： Re： [Sssssuperior/VSCode] VSOD数据集 （Issue [#11](https://github.com/Sssssuperior/VSCode/issues/11)） 感谢您的耐心等待！您能否上传带有轮廓和最佳流动的 DAVSOD 训练集的链接？百度上的链接似乎被破坏了。您可以通过 google drive 上传带有轮廓和最佳流的 DAVSOD 训练集的链接吗？感谢！！！https://drive.google.com/file/d/1ttb91Q5LzryPyV8uHwkTHmLLtynNk1pk/view?usp=sharing – 直接回复此电子邮件、在 GitHub 上查看或取消订阅。您收到此消息是因为您发表了评论。消息 ID：_**@**_.***>
> > 
> > 
> > 事实上，我在本地计算机上下载了提供的数据集，文件没有错误......您可以尝试使用 unzip repair 命令。这可能是文件格式的问题。我将尝试将其打包到一个新文件中，您可以查看此文件是否有效：https://drive.google.com/file/d/1wU0V8ePent79PZn4NtdBYreUcKvHju7b/view?usp=sharing
> 
> 您好，您提供的这个 DAVSOD 训练集链接似乎已经损坏。您能否发送一个可用的Google或百度链接？感谢！！！ [@Sssssuperior](https://github.com/Sssssuperior)

您好，可以试试这个文件可以吗：通过网盘分享的文件：DAVSOD.tar
链接: https://pan.baidu.com/s/12cIjBYPY6dolYldAAKGa3Q?pwd=p9b6 提取码: p9b6

**Comment by 777-qi** [comment]
DAVSOD.tar
链接: https://pan.baidu.com/s/12cIjBYPY6dolYldAAKGa3Q?pwd=p9b6 提取码: p9b6
压缩包损坏了，能否再提供一下
